### PR TITLE
[EASI-3928] - Updated IT Solution references

### PR DESCRIPTION
**Pull request by patrickseguraoddball**
# EASI-3928

## Changes and Description

- Updated `IT Solution` references per [doc](https://docs.google.com/document/d/1O2-F8THZ6cOm8zoEMiO__mn5xEwKCw3pgsdG_iXBO9M/edit)
- Updated snapshots

## How to test this change

Verify changes match doc

## PR Author Review Checklist

- [ ] Met the ticket's acceptance criteria, or will meet them in a subsequent PR.
- [ ] Added or updated tests for backend resolvers or other functions as needed.
- [ ] Added or updated client tests for new components, parent components, functions, or e2e tests as necessary.
- [ ] Updated the [Postman Collection](../MINT.postman_collection.json) if necessary.

## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.

### CODE REVIEW

File: src/i18n/en-US/modelPlanTaskList.ts [hunk 1]
```diff
@@ -97,9 +97,9 @@ const modelPlanTaskList = {
       path: 'payment'
     },
     itSolutions: {
-      heading: 'IT solutions and implementation status',
+      heading: 'Operational solutions and implementation status',
       team:
-        'Choose the IT solutions your model model will utilize. Many items in this section are populated based on responses to questions answered in previous sections.',
+        'Choose the solutions your model model will utilize. Many items in this section are populated based on responses to questions answered in previous sections.',
```
Comment (garyjzhao) [inline src/i18n/en-US/modelPlanTaskList.ts:102]: Copy is not correct. There is an extra "model"

> Choose the solutions your **model model** will util....

**Review by garyjzhao — CHANGES_REQUESTED** [review]
Just one change requested.

File: src/i18n/en-US/modelPlanTaskList.ts (hunk 1 above)
Comment (patrickseguraoddball) [reply src/i18n/en-US/modelPlanTaskList.ts:102]: Lol that has always been there, I didn't touch that code.  Nice catch though

**Review by garyjzhao — APPROVED** [review]
LGTM

**Pull request merged**